test/008b-rating.test.js

```diff
@@ -0,0 +1,168 @@
+import models from '../models';
+import server from '../app';
+
+const { User } = models;
+const baseUrl = '/api/v1/articles';
+
+chai.use(chaiHttp);
+
+describe('Test Suite for Rating', () => {
+  let nonMentorToken;
+  let isMentorToken;
+  let secondUserToken;
+  before(async () => {
+    await models.sequelize.sync({ force: true });
+
+    const firstUserRequestObject = {
+      username: 'bugsburney',
+      email: 'bugsburney@email.com',
+      password: 'bugsbugs'
+    };
+    const secondUserRequestObject = {
+      username: 'daffyduck',
+      email: 'daffyduck@email.com',
+      password: 'daffydaffy'
+    };
+    const firstArticleRequestObject = {
+      title: 'Silicon Valley',
+      body: 'Article Body',
+      description: 'Article Description',
+      category: 'Science'
+    };
+    const secondArticleRequestObject = {
+      title: 'Tales of Two Strangers',
+      body: 'Article Body',
+      description: 'Article Description',
+      category: 'Science'
+    };
+    const firstLoginRequestObject = {
+      usernameOrEmail: 'bugsburney',
+      password: 'bugsbugs'
+    };
+    const secondLoginRequestObject = {
+      usernameOrEmail: 'daffyduck',
+      password: 'daffydaffy'
+    };
+
+    const nonMentorResponseObject = await chai.request(server)
+      .post('/api/v1/users')
+      .send(firstUserRequestObject);
+    nonMentorToken = nonMentorResponseObject.body.token;
+
+    await chai.request(server)
+      .post('/api/v1/users')
+      .send(secondUserRequestObject);
+
+    await chai.request(server)
+      .post(baseUrl)
+      .send(firstArticleRequestObject)
+      .set('Authorization', nonMentorToken);
+
+    const firstUser = await User.findOne({ where: { username: 'bugsburney' } });
+    await firstUser.update({
+      isMentor: true,
+      password: 'bugsbugs'
+    });
+
+    const secondUser = await User.findOne({ where: { username: 'daffyduck' } });
+    await secondUser.update({
+      isMentor: true,
+      password: 'daffydaffy'
+    });
+
+    const isMentorResponseObject = await chai.request(server)
+      .post('/api/v1/users/login')
+      .send(firstLoginRequestObject);
+    isMentorToken = isMentorResponseObject.body.token;
+
+    const secondUserResponseObject = await chai.request(server)
+      .post('/api/v1/users/login')
+      .send(secondLoginRequestObject);
+    secondUserToken = secondUserResponseObject.body.token;
+
+    await chai.request(server)
+      .post(baseUrl)
+      .send(secondArticleRequestObject)
+      .set('Authorization', secondUserToken);
+
+    await chai.request(server)
+      .patch(`${baseUrl}/rating/tales-of-two-strangers-daffyduck`)
+      .set('Authorization', secondUserToken)
+      .send({ rateValue: '2' });
+  });
+
+  describe('Rate Article', () => {
+    it('should not rate an article if user is not logged in',
+      async () => {
+        const response = await chai.request(server)
+          .patch(`${baseUrl}/rating/silicon-valley-bugsburney`)
```

Comment:
You should get the `slug` from the response of the `create article` request. Don't hardcode it.